### DISCUSSION TITLE
QR format

### DISCUSSION THREAD

**Discussion by jquiros2** [btcpayserver/btcpayserver#2921] [General]
Hi!  I finally figured out why a wallet (chivo) was not liking addresses.  It is because it can only read "bitcoin:bech32" (both lowercase) addresses in the QR.  See https://github.com/btcpayserver/btcpayserver/issues/2110 for reference.

Right now btcpay server is making them lower:UPPER.

Since I do not know if the wallet developers will update this anytime soon, what might the quickest way to make btcpay generate them as "lower:lower"

I am using btcpayserver-docker

**Comment by bolatovumar** [comment]
@jquiros2 that's a bug with Chivo. I remember reading somewhere that they were fixing it.

**Reply by jquiros2** [reply]
Yeah, they say a lot.  As of today, still no fix.  FYI for anyone interested there is a "chivo switch" fork for btcpayserver I am keeping pretty current.  It uses some code provided by @NicolasDorier for backward compatibility with chivo.  Though chivo really has otrher larger issues (pending transactions that are discounted from the user's balance but never make it through).
